features/cloud.py

```diff
@@ -552,64 +428,61 @@ def _create_instance(
         )
         inst = self.api.launch(
             image_id=image_name,
+            instance_type="Standard_DC1s_v3",
```

Comment:
Let's try with `Standard_A2_v2`. https://docs.microsoft.com/en-us/azure/virtual-machines/av2-series says

> Av2-series VMs have CPU performance and memory configurations best suited for entry level workloads like development and test

and:

> [VM Generation Support](https://docs.microsoft.com/en-us/azure/virtual-machines/generation-2): Generation 1